### ISSUE TITLE
[FEATURE] Theme Support

### ISSUE THREAD

**Issue by Coccocoahelper** [CCBlueX/LiquidBounce#2899]
### LiquidBounce Branch

Legacy

### Describe your feature request.

Allow theme installing and loading similar to how configs are done (Local and Cloud), allowing you to pick either visual (subjective) modules and settings, HUD elements, or both.

**Comment by Razzy52** [comment]
can already be done locally
https://github.com/razzy52/liquidthemes

**Comment by EclipsesDev** [comment]
> can already be done locally https://github.com/razzy52/liquidthemes

pretty sure thats user custom themes.. he want a cloud based pre made theme in legacy

**Comment by EclipsesDev** [comment]
for locally ig thats easy to do

**Comment by Coccocoahelper** [comment]
> can already be done locally https://github.com/razzy52/liquidthemes

Legacy, not Nextgen...

**Comment by EclipsesDev** [comment]
> > can already be done locally https://github.com/razzy52/liquidthemes
> 
> Legacy, not Nextgen...

I'm pretty sure themes existed in legacy.. but I'm unsure.. kinda forgot

**Comment by EclipsesDev** [comment]
yep its kinda exist https://github.com/CCBlueX/LiquidBounce/tree/legacy/src/main/java/net/ccbluex/liquidbounce/ui/client

https://github.com/CCBlueX/LiquidBounce/tree/legacy/src/main/java/net/ccbluex/liquidbounce/ui/client/clickgui/style/styles

**Comment by Coccocoahelper** [comment]
> yep its kinda exist https://github.com/CCBlueX/LiquidBounce/tree/legacy/src/main/java/net/ccbluex/liquidbounce/ui/client
> 
> https://github.com/CCBlueX/LiquidBounce/tree/legacy/src/main/java/net/ccbluex/liquidbounce/ui/client/clickgui/style/styles

You can't save or load them, only with a very complex method.

**Comment by Coccocoahelper** [comment]
Local Themes ready, only the cloud themes are missing.

**Comment by EclipsesDev** [comment]
> Local Themes ready, only the cloud themes are missing.

Yeah, I'll add it soon ig

**Comment by 1zun4** [comment]
https://github.com/CCBlueX/LiquidBounce/pull/5446
